library/tomee

```diff
@@ -1,67 +1,52 @@
-Maintainers: Alex Soto <asotobu@gmail.com> (@lordofthejars), Otavio Santana <otaviojava@apache.org> (@otaviojava), Jonathan Gallimore <jgallimore@apache.org> (@jgallimore)
+Maintainers: Alex Soto <asotobu@gmail.com> (@lordofthejars), Otavio Santana <otaviojava@apache.org> (@otaviojava), Jonathan Gallimore <jgallimore@apache.org> (@jgallimore), Rod Jenkins <jenkir14@nationwide.com> (@scriptmonkey)
```

Comment:
Ping image Maintainers Alex Soto (@lordofthejars), Otavio Santana (@otaviojava), Jonathan Gallimore (@jgallimore) about adding Rod Jenkins (@scriptmonkey) to Tomee image maintainers list.  Can one of you confirm?

Reply:
@yosifkit Yes - please add Rod as a maintainer. Many thanks!

Reply:
That's ok to me.